crates/fixedpointmath/src/lib.rs

```diff
@@ -1,859 +1,35 @@
+//! A generic fixed point wrapper around the `U256` type from ethers-rs.
```

Comment:
Since `ethers-rs` is being deprecated, I checked out alloy to see what they use for ints now and they're using [ruint](https://crates.io/crates/ruint) which looks nice. Also found [uint](https://crates.io/crates/uint). It might be nice to refactor to something similar soon.